docs/SCM-BestPractices/github/README.md

```diff
@@ -1,91 +1,61 @@
-# GitHub Configuration Best Practices
+# Table of Content
 
-## Intro
+## Continuous Integration / Continuous Deployment
```

Comment:
Intro section is missing. we should consider how to avoid losing that as we roll in new updates.